README.md

```diff
@@ -0,0 +1,89 @@
+# DACCS Docker image updater
+
+
+STATUS : Experimental
+
+Automatically updates `birdhouse-deploy` docker images as soon as new tags are pushed.
+
+## Requirements
+
+- hub (via brew)
+- jq
+
+.. and for tests ..
+
+- python3
+
+
+# Usage
+
+## Command line
+
+```
+# remove `EXIT_BEFORE_PR=1` if you want to actually create the PR
+EXIT_BEFORE_PR=1 GITHUB_USER=XXXX GITHUB_PASSWORD=YYYY ./main.sh
+```
+
+## Docker
+
+```
+chmod +x ./Taskfile
+
+# remove `EXIT_BEFORE_PR=1` if you want to actually create the PR
+HISTORIC_TAG_DATA_PATH=/absolute-and-writable-path EXIT_BEFORE_PR=1 GITHUB_USER=XXXX GITHUB_PASSWORD=YYYY ./Taskfile build-run
```

Comment:
I really like the `Taskfile`.  I think this Docker section and Taskfile should be before the "Command line" section and the Requirements section be moved to a sub-section of the "Command line" section.

Reply:
Good idea!